community/contributing/docs/docs-style-guide.md

```diff
@@ -0,0 +1,292 @@
+| This guide teaches you how to use Plugins. | In this guide, we are going to learn about Plugins.|
+
+### Avoid jargon and idioms
+
+Avoid jargon and idioms to help non-native English speakers understand the content better.
+
+|**Do** |**Do Not**|
+|:------|:---------|
+| Internally, ...| Under the hood, ...|
+| Create a new instance. | Spin up a new instance.|
+
+### Avoid statements about the future
+
+If you need to write about an alpha feature, put the text in a note or under a heading that identifies it as alpha information.
+
+### Avoid statements that will soon be out of date
+
+Avoid using words like "currently" and "new." A feature that is new today might not be considered new in a few months.
+
+|**Do** |**Do Not**|
+|:------|:---------|
+| In version 1.4, ... | In the current version, ...|
+| The Plugins feature provides ... | The new Plugin feature provides ...|
+
+## Documentation formatting standards
+
+### Use sentence capitalization for headings
+
+~~~~~~~~~
+# Creating a custom webhook stage
+
+## Configuring parameters for custom webhook stages
+~~~~~~~~~
+
+### Line breaks
+U#se a single newline to separate block-level content like headings, lists, images, code blocks, paragraphs, and others.
+
+### Use camel case for API objects
+
+Use the same uppercase and lowercase letters that are used in the
+actual object name when you write about API objects. The names of API objects use [Camel case](https://en.wikipedia.org/wiki/Camel_case).
+
+Don't split the API object name into separate words. For example, use CredentialsController, not Credentials Controller.
+
+Refer to API objects without saying "object," unless omitting "object" leads to an awkward construction.
+
+|**Do** |**Do Not**|
+|:------|:---------|
+| The PipelineController restarts a Stage. | The pipeline controller restarts a stage.|
+| The AmazonInfrastructureController is responsible for ... | The AmazonInfrastructureController object is responsible for ...|
+
+### Use angle brackets for placeholders
+
+Use angle brackets for placeholders. Describe what a placeholder represents.
+
+For example:
+`hal plugins repository add <unique-repo-name> --url <repo-url>`
+
+
+### Use bold for user interface elements
+
+|**Do** |**Do Not**|
+|:------|:---------|
+| Click **Fork**.| Click "Fork".|
+| Select **Other**. | Select "Other".|
+
+
+### Use italics to define or introduce new terms
+
+|**Do** |**Do Not**|
+|:------|:---------|
+|A _Stage_ is a step in a pipeline ... | A "Stage" is a step in a pipeline ...|
+
+### Use code style for filenames, directories, and paths
+
+|**Do** |**Do Not**|
+|:------|:---------|
+| Open the `rosco.yaml` file. | Open the rosco.yaml file.|
+| Go to the `/docs/tutorials` directory. | Go to the /docs/tutorials directory.|
+| Open the `/.hal/config` file. | Open the /.hal/config file.|
+
+### Lists
+Group items in a list that are related to each other. Use a numbered list for items that need to appear in a specific order.
```

Comment:
Better would be: Use a numbered list for instructions that need to be completed in a specific order.